prusa/connect/printer/util.py

```diff
@@ -12,7 +13,13 @@
 def get_timestamp(timestamp: float = None):
     """If given None, gets the current timestamp, otherwise leaves the
     value alone"""
-    return timestamp or int(time() * 10) * const.TIMESTAMP_PRECISION
+    result = timestamp or int(time() * 10) * const.TIMESTAMP_PRECISION
+    return result
+
+
+def make_fingerprint(identifier):
```

Comment:
It looks like identifier has to be str, use mypy type here please

Reply:
whoops, you're right